Documentation/Index.rst

```diff
@@ -71,32 +57,27 @@ TCA properties change between major versions, this main array had significant re
 especially between core versions 6.2 to 7 LTS, and again between 7 LTS and 8 LTS. Use the version
 selector of this documentation to select the documentation variant that fits.
 
+.. _credits:
 
 **Credits**
 
-The original reference to the TCA was written by Kasper Skårhøj. This
-version has been updated by François Suter. Lots of core team members and
-other contributors maintain this document over time, see
-`github <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi>`__ for
-an overview of recent contributors. A big "Thank You!" goes to every single one of them.
+The original reference to the TCA was written by Kasper Skårhøj. Subsequent
+versions have been updated by François Suter. Several core team members and
+other contributors maintained this document over time, see the list of
+`contributors on GitHub <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/graphs/contributors>`__.
 
+A big "Thank You!" goes to every single one of them.
 
-**Feedback**
 
-For general questions about the documentation get in touch by writing
-to `documentation@typo3.org <mailto:documentation@typo3.org>`_ .
+**Contribution**
 
-If you find a bug in this manual, please be so kind as to check the
-`online version <https://docs.typo3.org/typo3cms/TCAReference/>`__.
-Use the "Edit me on GitHub" button in the top right corner
-and submit a pull request via GitHub. Alternatively you can just file an issue
-using the `bug tracker <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/issues>`__.
+For general questions about the documentation get in touch with the
+`Documentation Team <https://typo3.org/community/teams/documentation>`__.
 
-Maintaining high quality documentation requires time and effort
-and the TYPO3 Documentation Team always appreciates support.
-If you want to support us, please join the `#typo3-documentation` channel
-on `typo3.slack.com <https://typo3.slack.com/>`__ to get in contact. If you're not
-using Slack yet, trigger a `Slack invite <https://forger.typo3.com/slack>`__.
+If you find a bug in this manual, please be so kind as to make a change, e.g.
+by using the "Edit me on GitHub" button in the top right corner.
```

Comment:
Do we have/need an annotation/emphasis for the button text?

Reply:
Do you mean a different style. Correct would be `:guilabel:`, at least that is what we usually use.

Reply:
Then we should use this. :-)

Reply:
Ok, coming up.